google/cloud/spanner/read_partition.cc

```diff
@@ -36,6 +36,45 @@ ReadPartition::ReadPartition(std::string transaction_id, std::string session_id,
   *proto_.mutable_key_set() = spanner_internal::ToProto(std::move(key_set));
   proto_.set_limit(read_options.limit);
   proto_.set_partition_token(std::move(partition_token));
+  if (read_options.request_priority) {
+    auto* request_options = proto_.mutable_request_options();
+    switch (*read_options.request_priority) {
+      case spanner::RequestPriority::kLow:
+        request_options->set_priority(
+            google::spanner::v1::RequestOptions_Priority_PRIORITY_LOW);
```

Comment:
I think the intended enum name in the generated code is `google::spanner::v1::RequestOptions::PRIORITY_LOW`:

https://developers.google.com/protocol-buffers/docs/reference/cpp-generated#enum

the additional constants `...RequestOptions_Priority_PRIORITY_LOW` are not defined in the spec and (I would guess) can change.

Reply:
I'll adjust the naming in a followup PR.